tableschema/types/array.py

```diff
@@ -4,42 +4,21 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import six
 import json
-from .. import exceptions
-from .. import helpers
-from . import base
+from ..config import ERROR
 
 
 # Module API
 
-class ArrayType(base.JTSType):
-
-    # Public
-
-    name = 'array'
-    null_values = helpers.NULL_VALUES
-    supported_constraints = [
-        'required',
-        'pattern',
-        'enum',
-        'minLength',
-        'maxLength',
-    ]
-    # ---
-    python_type = list
-
-    def cast_default(self, value, fmt=None):
-
-        if isinstance(value, self.python_type):
-            return value
-
+def cast_array(format, value):
```

Comment:
This interface is totally internal so I've selected this only to make tests - https://github.com/frictionlessdata/tableschema-py/blob/null-types-constraints-v1/tests/types/test_datetime.py - to look more like DSL (I mean invocation consistency - type -> format -> value). But for end-user there is no difference.